include/net/net_pkt.h

```diff
@@ -2292,9 +2292,9 @@ int net_pkt_memset(struct net_pkt *pkt, int byte, size_t length);
 /**
  * @brief Copy data from a packet into another one.
  *
- * Note: Both net_pkt cursors should be properly initialized and,
- *       eventually, properly positioned using net_pkt_skip.
- *       Cursors will be updated according to parameters.
+ * Note: Both net_pkt cursors should be properly initialized and, if needed,
+ *       positioned using net_pkt_skip.
+ *       The cursors will be updated after the operation.
```

Comment:
s/cursors/cursor

s/after/during  (the cursor is used and updated for each partial moves, if there are partial move: when it's not contiguous for instance)

Reply:
> s/cursors/cursor

Here two cursors are used, so plural is right.

> s/after/during (the cursor is used and updated for each partial moves, if there are partial move: when it's not contiguous for instance)

But that's implementation detail. What is of interest to the user is the effect of the operation, i.e. the state after the operation is called. That's why the word "after" is used.

Reply:
Ok, to avoid any confusion, will use "Cursor position will be updated after the operation."

Reply:
ok